File: WebApp/Scripts/app/model/DGMISS_MAST.js
Ext.define('WEBAPP.model.DGMISS_MAST', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'INID', type: 'string' },
        { name: 'INID_T', type: 'string' },
        { name: 'MMCODE', type: 'string' },
        { name: 'MMNAME_C', type: 'string' },
        { name: 'MMNAME_E', type: 'string' },
        { name: 'CREATE_TIME', type: 'string' },
        { name: 'CREATE_USER', type: 'string' },
        { name: 'UPDATE_TIME', type: 'string' },
        { name: 'UPDATE_USER', type: 'string' },
        { name: 'UPDATE_IP', type: 'string' }
    ]
});

File: WebApp/Models/D/DGMISS_MAST.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApp.Models.D
{
    public class DGMISS_MAST : JCLib.Mvc.BaseModel
    {
        public string INID { get; set; }
        public string INID_T { get; set; }
        public string MMCODE { get; set; }
        public string MMNAME_C { get; set; }
        public string MMNAME_E { get; set; }
        public string CREATE_TIME { get; set; }
        public string CREATE_USER { get; set; }
        public string UPDATE_TIME { get; set; }
        public string UPDATE_USER { get; set; }
        public string UPDATE_IP { get; set; }
    }
}